src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystem.java

```diff
@@ -263,7 +263,7 @@ private static void createBranchActivePowerTargetEquation(LfBranch branch, Discr
 
     private static void createDiscreteVoltageControlEquation(LfBus bus,  VariableSet variableSet, EquationSystem equationSystem) {
         if (bus.isDiscreteVoltageControlled()) {
-            EquationTerm vTerm = EquationTerm.createVariableTerm(bus, VariableType.BUS_V, variableSet, bus.getV().eval());
+            EquationTerm vTerm = EquationTerm.createVariableTerm(bus, VariableType.BUS_V, variableSet, bus.getDiscreteVoltageControl().getTargetValue());
```

Comment:
@geofjamg I think it is ` bus.getV().eval()`

Reply:
Using a numeric value of a state variable in the equation system is not correct.
bus.getDiscreteVoltageControl().getTargetValue() is better, but not correct too. It should be redesign to move it in the RHS.

Reply:
After thinking again, I am not sure to understand why do we need to set an initial value for equation term at this place. Normally equation initial value should be computed from initial state vector. To investigate before merging this PR.

Reply:
Yes indeed I understand now the purpose of this new term and we have to investigate why we need it now.

Reply:
The initial value is only used if the voltage initializer type is set to `PREVIOUS_VALUES`. (because it causes a call to lfBus.getV().eval() ). Otherwise it is indeed replaced immediately by the value from the state vector.

Reply:
But with PREVIOUS_VALUES the bus voltage is initialized from IIDM values (if defined). Why do you need to override this value?

Reply:
The PREVIOUS_VALUES was initialized with LfBus.getV() (now lfBus.getV().eval()). The issue is that after the creation of the system, lfBus.getV() returns the EquationTerm representing V in the system, and not the value found in the IIDM, so we need to pass the value from the IIDM to the EquationTerm so it does not get lost. (or we could also try to call the voltage initialization before the creation of the equation system)

Reply:
Ok I understand, so @annetill is right in her comment, it should be bus.getV().eval() in that case to really init from IIDM network state as it was before.